src/utils/time-entry-utils/timeEntryUtils.js

```diff
@@ -32,3 +32,15 @@ export const removeTimecardContextEntry = (
   newTimeEntries.splice(removeAtIndex, 1);
   setTimeEntries(newTimeEntries);
 };
+
+export const calculateFinishTimeOnNextDay = (
+  startTimeValue,
+  finishTimeValue
+) => {
+  if (startTimeValue && finishTimeValue) {
+    return Math.floor(
```

Comment:
I'm assuming the purpose of this function is to determine if the finish time is on the next day.
It looks like we're doing some sort of manual conversion of the time values here to get that result.
If so, we should make use of the dayJs library to greatly simplify this operation and return a boolean instead.